microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileCompletions.java

```diff
@@ -95,11 +97,29 @@ public CompletionList doComplete(PropertiesModel document, Position position, Mi
 		case COMMENTS:
 			// no completions
 			break;
+
+		case PROPERTY_VALUE_EXPRESSION:
+			PropertyValueExpression propExpr = (PropertyValueExpression) node;
+			if (offset == propExpr.getStart() || (propExpr.isClosed() && propExpr.getEnd() == offset)) {
+				collectPropertyValueSuggestions(node, document, projectInfo, valuesRulesManager, completionSettings, list);
+			} else {
+				collectPropertyValueExpressionSuggestions(node, document, projectInfo, valuesRulesManager, completionSettings, list);
+			}
+			break;
+
 		case ASSIGN:
+			// Only collect if on right side of =
+			if (offset == node.getEnd()) {
+				collectPropertyValueSuggestions(node, document, projectInfo, valuesRulesManager, completionSettings, list);
+			}
+			break;

```

Comment:
This is a fix for https://github.com/eclipse/lsp4mp/issues/14; should I move it to a separate PR?

Reply:
If we have tests for this case, I think it makes the most sense to move the fix and the tests to a separate PR, since the scope of this PR is already quite big.

Reply:
I took out the code from this PR and will make a separate PR for it.